after_deployment_tasks.md

```diff
@@ -13,6 +13,14 @@ Check that you now
 
 * see "1+2=3" in each app.
 * have been redirected to https
+
+Setup cloudflare: https://www.cloudflare.com/cloudflare-settings?z=renuoapp.ch
```

Comment:
This direct link does not work for me :/ Maybe https://www.cloudflare.com/a/overview/renuoapp.ch ?